pwiz_tools/Skyline/Util/Util.cs

```diff
@@ -2001,6 +2028,14 @@ public static void AreEqual(double expected, double actual, double delta, string
 
         public static void Fail(string error = "")
         {
+            if (InvokeDebuggerOnFail)
+            {
+                Console.WriteLine();
+                if (!string.IsNullOrEmpty(error))
+                    Console.WriteLine(error);
+                Console.WriteLine(@"error encountered, launching debugger as indicated by Assume.MachineNamePatternForInvokeDebuggerOnFail");

```

Comment:
This text is no longer valid.

Reply:
Right, thanks. I'll clean that up then merge to master.